### PR TITLE
Add media instructions

### PR DESCRIPTION
**Pull request by zjesko**
Added documentation as to how to include media in the evaluation object

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,23 +79,35 @@ class ExampleEvaluator:
...
+    
+    media_dir = '/tmp/'
+
+    """
+    To add media to the result object such that it shows on the challenge leaderboard:
+    - Simply save the file at '/tmp/<filename>'
+    - Add the path of the media to the result object:
+        For images, add file path to _result_object["media_image_path"]
+        For videos, add file path to _result_object["media_video_path"]
+    """
```
Comment (nemaniarjun) [inline README.md]: Maybe save an image to `/tmp/filename` and add it to the `_result_object` as intended.

Comment (nemaniarjun) [reply README.md]: any photo would do

Comment (nemaniarjun) [reply README.md]: And also about thumbnail images,  and how to add them

Comment (nemaniarjun) [reply README.md]: Example things that you can do here
```python
   [1]:from PIL import Image, ImageDraw 
   ...:   
   ...: img = Image.new('RGB', (300, 30), color = (73, 109, 137)) 
   ...:   
   ...: d = ImageDraw.Draw(img) 
   ...: d.text((10,10), "AIcrowd Example Leaderboard Image :)", fill=(255,255,0)) 
   ...:   
   ...: img.show()                                                                                                                                 

```

Comment (akshatcx) [reply README.md]: basically add an example right?

**Comment by nemaniarjun** [comment]
Yep

**Review by skbly7 — APPROVED** [review]
LGTM

**Pull request merged**